src/main/java/com/doongji/nestalk/controller/v1/user/UserRestController.java

```diff
@@ -59,4 +58,12 @@
         String email = userService.findEmailByNameAndPhone(request.getName(), request.getPhone());
         return ResponseEntity.ok(new FindEmailResponse(email));
     }
+
+    @ApiOperation(value = "회원탈퇴")
+    @DeleteMapping(path = "user/delete")
```

Comment:
```suggestion
    @DeleteMapping(path = "user")
```

DELETE Method를 썼다면 굳이 url에 delete를 또 추가할 필요는 없을 것 같아요.